packages/editor/editor.css

```diff
@@ -472,6 +473,10 @@ input,
 	stroke-width: calc(2.5px * var(--tl-scale));
 }
 
+.tl-custom-overlays {
```

Comment:
"custom" here sorta implies that it's not being used by tldraw and you can use it as an escape hatch for SDK users (but it will be used by elbow arrows). similarly, it implies that the other layers aren't customizable heh.

i'd just nix the "custom" part of this.

Reply:
`tl-overlays` already exists! it contains a bunch of pre-defined things like selection fg/bg, handles, indicators, etc. This is like a way of getting something extra into the overlays? I couldn't think of a better name but open to other suggestions!